hebrew/bar/bar.md

```diff
@@ -7,4 +7,8 @@
 
 עישנת לעצמך איזה ג'וינט טוב.
 
-מה עכשיו?
\ No newline at end of file
+מה עכשיו?
+
+[תתקשר לאסתי גינזבורג](hebrew/esti/esti.md) 
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.